config/config.go

```diff
@@ -131,7 +131,7 @@ type SourceConfig struct {
 	// Nodes describe the oasis-node(s) to connect to. Keys are "archive
 	// names," which are named after mainnet releases, in lowercase e.g.
 	// "cobalt" and "damask."
-	Nodes map[string]*NodeConfig `koanf:"nodes"`
+	Nodes map[string]*ArchiveConfig `koanf:"nodes"`
```

Comment:
Remember to update the checked-in .yml files accordingly.

Reply:
:scream: the sample files!